.travis.yml

```diff
@@ -52,13 +52,13 @@ matrix:
     bundler_args: --without changelog development docgen guard integration maintenance omnibus_package tools aix bsd mac_os_x solaris windows --frozen
   - env:
       CHEFSTYLE: 1
-    rvm: 2.1
+    rvm: 2.2.5
```

Comment:
I just wonder if we'd be better off with them at 2.3.1 since thats what we plan to roll with.